console-app/EventSource-ConsoleApp.csproj

```diff
@@ -38,12 +37,6 @@
     <Reference Include="System.Core" />
     <Reference Include="System.IO.Compression" />
     <Reference Include="System.Numerics" />
-    <Reference Include="System.Runtime.CompilerServices.Unsafe, Version=4.0.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">

```

Comment:
This one was unused and VS was complaining about the version number mismatch here.
I noticed it when doing an audit of DLLs that we ship at my company.

Reply:
I'm a bit confused about that last part since this project isn't part of anything that would be shipped; it's strictly for manual testing of the library. The dependency in question does seem to be unnecessary, but I'm not sure how it would have any effect on what you're doing.

Reply:
Though I am leaning toward just getting rid of this test app—I think it was only in there because at first we didn't have any real end-to-end HTTP in the unit tests.